### PR TITLE
It_it - italian translation

### PR DESCRIPTION
**Pull request by Lorenzo0310200**

### CODE REVIEW

**Comment by Lorenzo0310200** [comment]
By Lorenzo Giannini

File: src/main/resources/assets/freecam/lang/it_it.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "text.autoconfig.freecam.option.horizontalSpeed": "Velocità Orizzontale",
+  "text.autoconfig.freecam.option.verticalSpeed": "Velocità Verticale",
+  "text.autoconfig.freecam.option.noclip": "No Clip",
+  "text.autoconfig.freecam.option.freezePlayer": "Blocca Giocatore",
+  "text.autoconfig.freecam.option.showPlayer": "Mostra Giocatore",
+  "text.autoconfig.freecam.option.showHand": "Mostra Mano",
+  "text.autoconfig.freecam.option.disableOnDamage": "Disattiva Danno",
+  "text.autoconfig.freecam.option.allowInteract": "Consenti Interazione",
+  "text.autoconfig.freecam.option.notifyFreecam": "Notifiche Camera libera",
+  "text.autoconfig.freecam.option.notifyPersistent": "Notifiche Persistenti"
```
Comment (Lorenzo0310200) [inline src/main/resources/assets/freecam/lang/it_it.json:21]: here I have translated persistent notifications. I do not know what it means.

**Comment by hashalite** [comment]
Thanks for translating!

**Comment by Lorenzo0310200** [comment]
> Thanks for translating!

👍

**Pull request merged**